community/blog/2022-05 New Azure backround jobs and Managed Identity support in Arcus.BackgroundJobs v0.4/README.md

```diff
@@ -0,0 +1,95 @@
+
+This overview gives you an idea of this workflow:
+![Arcus App Configuration Azure setup](media/app-config-background-job.png)
+
+The job is available in the new library called: `Arcus.BackgroundJobs.AppConfiguration`. Note that this job requires the [Arcus secret store](https://security.arcus-azure.net/features/secret-store/) to interact with the Azure App Configuration and a registered App Configuration configuration source within your application configuration registration so a specific value can be updated.
+
+```csharp
+using Microsoft.Extensions.DependencyInjection;
+
+public class Program
+{
+    public static IHostBuilder CreateHostBuilder()
+    {
+        return Host.CreateDefaultBuilder()
+                   .ConfigureSecretStore(...)
+                   .ConfigureAppConfiguration(builder =>
+                   {
+                       builder.AddAppConfiguration(options =>
+                       {
+                           options.Connect("<connection-string>")
+                                  .Register("<configuration-key>");
+                       });
+                   })
+                   .ConfigureServices(services =>
+                   {
+                       services.AddAutoRefreshAppConfigurationBackgroundJob(
+                            //    this allows the background jobs to support applications that are running multiple instances, processing the same type of events, without conflicting subscription names.
+                            subscriptionPrefix: "TestSub",
+                            // Connection string secret key to a Azure Service Bus Topic.
+                            serviceBusTopicConnectionStringSecretKey: "MySecretKeyToServiceBusTopicConnectionString");
+                       );
+                   });
+    }
+}
+```
+
+For more information on this background job, see [our official documentation](https://background-jobs.arcus-azure.net/Features/AzureAppConfiguration/auto-refresh-app-configuration).
+
+## Managed Identity support for CloudEvents background jobs
+We are very happy to announce that the v0.4 release of BackgroundJobs also includes a valuable contribution that solves a missing functionality problem in the background jobs environment. This library heavily builds on top of our Arcus Messaging library. The Messaging library received some previous releases ago a feature that allowed Managed Identity authentication when registering message pumps. As Managed Identity is the way to go from a security perspective, this could not be missing in our BackgroundJobs library.
```

Comment:
```suggestion
We are very happy to announce that the v0.4 release of BackgroundJobs also includes a valuable contribution that solves a missing functionality problem in the background jobs environment. This library heavily builds on top of our Arcus Messaging library. The Messaging library received a feature some releases ago that allowed Managed Identity authentication when registering message pumps. As Managed Identity is the way to go from a security perspective, this could not be missing in our BackgroundJobs library.
```

Reply:
Ah, yes, that is better. Thx!